contents/docs/user-guides/trends.mdx

```diff
@@ -1,58 +1,324 @@
+-   `OR` - Events only have to match _a single_ filter within the group
+
+![example global filter](../../images/docs/user-guides/trends/filter-groups.png)
+
+One thing to note about global filters: since they apply to all events within an insight, and these events may not share all their properties, global filters allow you to use any property that has been seen in _at least one_ of your events.
+If you happen to create a filter that references a property that not all events have, events that do not contain this property will automatically fail the check and will be excluded by the filter.
+
+#### Filtering internal and test users
+
+The final option for filtering down events is using the option to filter out 'internal and test users'.
+This option is useful for excluding events sent from local builds of your product, or by members of your team that you don't want to include in your analysis.
+
+![filtering internal and test users turned on](../../images/docs/user-guides/trends/internal-and-test-users.png)
+
+If you want to customize the filters that are used, you can click on the settings icon.
+
+## Breaking down by properties
+
+Breakdowns offer a way for you to split up your trends graph by the values of a certain property. This is a very powerful tool when analyzing a trend and trying to determine the cause of a change in one of your metrics.
+Once you've selected a property to break down on, you should see your graph split into multiple lines, each one corresponding to a distinct value of that property.
+
+![breaking down an insight by browser](../../images/docs/user-guides/trends/breakdown.png)
+
+For readability, PostHog will only load the first 25 values of a breakdown when you first load an insight. You can manually load more values by clicking the 'Load more breakdown values' button.
+
+![graph of an insight broken down by a property](../../images/docs/user-guides/trends/breakdown-graph.png)
+
+> **Note: ** Currently, you're only able to breakdown an insight by a single property, but we're actively working on the ability to breakdown based on more than one property
+
+#### Breaking down numerical values into bins
+
+When breaking down based on a string value, your graph will be split up with separate series for _every distinct value_ of that property. This works well for text values, however when trying to analyze numeric or time properties,
+this is often far too granualar. As a solution, when breaking down a numeric property, you can select a number of 'Bins' to combine ranges of values into.
+
+![breaking down into 10 bins](../../images/docs/user-guides/trends/breakdown-into-bins.png)
+
+PostHog will then break up your events into the specified number of bins based on the distinct values of the property.
+For example, if your property can take the values `20, 21, 22, ..., 40` and you split it up into `10` bins, it will break your events up into ranges of `20 - 22`, `22 - 24`, etc.
```

Comment:
```suggestion
For example, if your property can take the values `20, 21, 22, ..., 40` and you split it up into `10` bins, it will break your events up into ranges of `20 - 22`, `22 - 24`, etc.

> Note that this aggregates all breakdown values, not just the top 25 ones, like it happens for non-numeric values
```